source.json

```diff
@@ -0,0 +1,9 @@
+{
+    "type": "raw",
+    "stream": {
+        "type": "s3",
+        "url": "http://10.0.208.243:80/rbd/re",
+        "access_key": "test_rbd",
+        "secret_key": "test_rbd"
+    }
+}
```

Comment:
Please add a newline at the end of file

Reply:
Inclusion of that file was a mishap
Have removed it
sorry

Reply:
Marking as resolved as the file has been removed.
Please feel free to revert if I missed something.